docker-compose.dbs.yml

```diff
@@ -1,16 +1,14 @@
 version: '3'
+
 services:
   db:
-    image: supabase/postgres:14.1.0
+    image: supabase/postgres:14.1.0.105
```

Comment:
Is there postgres v15 already available?

Reply:
@abc3 Yes, but most projects are on pg v14 so we should stick with it for the time being.